# coding: utf-8

from the_tale.common.utils.exceptions import TheTaleError

class HeroException(TheTaleError):
    MSG = u'hero error'
